PWGMM/Mult/TableProducer/trackPropagation.cxx

```diff
@@ -98,11 +98,10 @@ struct AmbiguousTrackPropagation {
 
   void initCCDB(ExtBCs::iterator const& bc)
   {
-    LOG(info) << "INITIALIZING CCDB";
-
     if (runNumber == bc.runNumber()) {
       return;
     }
+    LOG(info) << "INITIALIZING CCDB";
```

Comment:
Probably not very useful, since ccdb queries have their own printouts. Additionally, when running on hyperloop you get the statistics of executed queries. In general, it is recommended to avoid non-informative printouts.

Reply:
Ok I removed it